### PR TITLE
[zh-cn] Update Learn/Accessibility main page

### PR DESCRIPTION
**Pull request by jasonren0403**
### Description

Add sidebar and update content in accessibility main page

### CODE REVIEW

File: files/zh-cn/learn/accessibility/index.md [hunk 1]
```diff
@@ -3,36 +3,61 @@ title: 无障碍
...
 
-## 另见
+## 参见
 
-- [现在开始构建无障碍的 web 应用](https://egghead.io/courses/start-building-accessible-web-applications-today) — 来自 Marcy Sutton 的一系列优质的视频教程。
-- [DequeUniversity 资源](https://dequeuniversity.com/resources/) — 包含代码示例，屏幕阅读参考和其他有用的资源。
-- [WebAIM 资源](http://webaim.org/resources/) — 包含向导，清单，工具和其他资源。
+- [现在开始构建无障碍的 web 应用](https://egghead.io/courses/start-building-accessible-web-applications-today)——来自 Marcy Sutton 的一系列优质的视频教程。
+- [Deque University 资源](https://dequeuniversity.com/resources/)——包含代码示例，屏幕阅读参考和其他有用的资源。
+- [WebAIM 资源](http://webaim.org/resources/)——包含指引、清单、工具和其他资源。
```
Comment (jasonren0403) [inline files/zh-cn/learn/accessibility/index.md:62]:
```suggestion
- [WebAIM 资源](https://webaim.org/resources/)——包含指引、清单、工具和其他资源。
```

File: files/zh-cn/learn/accessibility/index.md [hunk 2]
```diff
@@ -3,36 +3,61 @@ title: 无障碍
...
+如果你想成为一个 web 开发者，学习一些 HTML、CSS 和 JavaScript 是很有用的，但是仅仅使用这些技术是不够的，重要的是要学会如何**负责任**地使用这些技术，以便所有读者都可能在网上使用你的创作。为了达到这个目的，要遵循一般最佳实践（这些最佳实践已经被 [HTML](/zh-CN/docs/Learn/HTML)、[CSS](/zh-CN/docs/Learn/CSS) 和 [JavaScript](/zh-CN/docs/Learn/JavaScript) 相关的话题所证明）进行[跨浏览器测试](/zh-CN/docs/Learn/Tools_and_testing/Cross_browser_testing)，并且从项目最初阶段就把无障碍性考虑在内。在这个模块中我们将会详细讨论后者。
 
-为了充分理解这个模块，你至少需要读过 [HTML](/zh-CN/docs/Learn/HTML), [CSS](/zh-CN/docs/Learn/CSS), 和 [JavaScript](/zh-CN/docs/Learn/JavaScript) 各个话题的前两个模块，或者有可能的话，在你学习相关联的技术话题时也把无障碍相关的内容学习了，这样会更好。
+## 概述
 
-> **备注：** 如果在你所使用的设备上你不能建立自己的文件，你可以使用在线编码平台运行大多数样例代码，比如 [JSBin](http://jsbin.com/) 或者 [Glitch](https://glitch.com/).
+当有人将一个网站描述为“无障碍”时，他们的意思是，任何用户都可以使用其所有的功能和内容，无论用户是如何访问网络的——甚至特别是有身体或精神障碍的用户。
 
-## 参考指南
+- 键盘、鼠标和触摸屏用户，以及用户访问网络的任何其他方式，包括像 Alexa 和 Google Home 这样的屏幕阅读器等语音助手，都应该能够访问网站。
```
Comment (yin1999) [inline files/zh-cn/learn/accessibility/index.md:14]: 尽量使用“它”而不是“他”

```suggestion
- 键盘、鼠标和触摸屏用户，以及用户访问网络的任何其它方式，包括像 Alexa 和 Google Home 这样的屏幕阅读器等语音助手，都应该能够访问网站。
```

File: files/zh-cn/learn/accessibility/index.md [hunk 3]
```diff
@@ -3,36 +3,61 @@ title: 无障碍
...
+- 无论人们的听觉、视觉、身体或认知能力如何，应用程序都应该是可以理解和使用的。
+- 网站也不应该造成伤害：像运动这样的网络功能会导致偏头痛或癫痫发作。
```
Comment (yin1999) [inline files/zh-cn/learn/accessibility/index.md:16]: https://support.google.com/webdesigner/answer/7411704?hl=zh-Hans

```suggestion
- 网站也不应该造成伤害：像动画（motion）这样的 web 特性会导致偏头痛或癫痫发作。
```

File: files/zh-cn/learn/accessibility/index.md [hunk 4]
```diff
@@ -3,36 +3,61 @@ title: 无障碍
 slug: Learn/Accessibility
 ---
 
-如果你想成为一个 web 开发者，学习一些 HTML,CSS 和 JavaScript 是很有用的，但是仅仅使用这些技术是不够的，你应该更进一步拓展你的知识 — 你需要响应式地使用它们来最大程度地发展你的用户，并且使用这些技术时不能把任何一门排斥出去。为了达到这个目的，要遵循一般最佳实践（这些最佳实践已经被 [HTML](/zh-CN/docs/Learn/HTML), [CSS](/zh-CN/docs/Learn/CSS), 和 [JavaScript](/zh-CN/docs/Learn/JavaScript) 相关的话题所证明）,进行跨浏览器测试，并且从项目最初阶段就把易使用性考虑在内。在这个模块中我们将会详细讨论后者。
+{{LearnSidebar}}
 
-## 预备知识：
+如果你想成为一个 web 开发者，学习一些 HTML、CSS 和 JavaScript 是很有用的，但是仅仅使用这些技术是不够的，重要的是要学会如何**负责任**地使用这些技术，以便所有读者都可能在网上使用你的创作。为了达到这个目的，要遵循一般最佳实践（这些最佳实践已经被 [HTML](/zh-CN/docs/Learn/HTML)、[CSS](/zh-CN/docs/Learn/CSS) 和 [JavaScript](/zh-CN/docs/Learn/JavaScript) 相关的话题所证明）进行[跨浏览器测试](/zh-CN/docs/Learn/Tools_and_testing/Cross_browser_testing)，并且从项目最初阶段就把无障碍性考虑在内。在这个模块中我们将会详细讨论后者。
```
Comment (yin1999) [inline files/zh-cn/learn/accessibility/index.md:8]:
```suggestion
如果你想成为一个 web 开发者，学习一些 HTML、CSS 和 JavaScript 是很有用的，但是仅仅使用这些技术是不够的，重要的是要学会如何**负责任**地使用这些技术，以便所有读者都可能在网上使用你的创作。为了达到这个目的，要遵循一般最佳实践（这些最佳实践已经被 [HTML](/zh-CN/docs/Learn/HTML)、[CSS](/zh-CN/docs/Learn/CSS) 和 [JavaScript](/zh-CN/docs/Learn/JavaScript) 相关的主题所证明）进行[跨浏览器测试](/zh-CN/docs/Learn/Tools_and_testing/Cross_browser_testing)，并且从项目最初阶段就把无障碍性考虑在内。在这个模块中我们将会详细讨论后者。
```

File: files/zh-cn/learn/accessibility/index.md [hunk 5]
```diff
@@ -3,36 +3,61 @@ title: 无障碍
...
+>
+> #### 想要成为一名前端工程师？
+>
+> 我们汇总了一门课程，其中包含你实现成为前端工程师目标所需要的所有基本信息。
+>
+> [**从这里开始**](/zh-CN/docs/Learn/Front-end_web_developer)
+
+## 预备知识
+
+为了充分理解这个模块，你至少需要读过 [HTML](/zh-CN/docs/Learn/HTML)、[CSS](/zh-CN/docs/Learn/CSS) 和 [JavaScript](/zh-CN/docs/Learn/JavaScript) 各个话题的前两个模块，或者有可能的话，在你学习相关联的技术话题时也把无障碍相关的内容学习了，这样会更好。
```
Comment (yin1999) [inline files/zh-cn/learn/accessibility/index.md:34]:
```suggestion
为了充分理解这个模块，你至少需要读过 [HTML](/zh-CN/docs/Learn/HTML)、[CSS](/zh-CN/docs/Learn/CSS) 和 [JavaScript](/zh-CN/docs/Learn/JavaScript) 各个主题的前两个模块，或者有可能的话，在你学习相关联的技术话题时也把无障碍相关的内容学习了，这样会更好。
```

File: files/zh-cn/learn/accessibility/index.md [hunk 6]
```diff
@@ -3,36 +3,61 @@ title: 无障碍
...
+
+> **备注：** 如果在你所使用的设备上不能建立自己的文件，你可以使用在线编码平台运行大多数样例代码，比如 [JSBin](http://jsbin.com/) 或者 [Glitch](https://glitch.com/)。
+
+## 指南
 
 - [什么是无障碍？](/zh-CN/docs/Learn/Accessibility/What_is_accessibility)
-  - : 这篇文章开头很好地阐述了到底什么是'无障碍' — 这包括我们要考虑哪一群人以及为什么要考虑这些人，不同用户使用什么工具与网页交互，以及我们在网页开发流中如何构建可用性部分。
+  - : 这篇文章开头很好地阐述了到底什么是无障碍——这包括我们要考虑哪一群人、为什么要考虑这些人、不同用户使用什么工具与网页交互、以及我们在网页开发流中如何构建无障碍部分。
```
Comment (yin1999) [inline files/zh-cn/learn/accessibility/index.md:41]: “以及”前面用了顿号。后面的内容请再仔细检查一下

**Review by yin1999 — CHANGES_REQUESTED** [review]

File: files/zh-cn/learn/accessibility/index.md (hunk 6 above)
Comment (jasonren0403) [reply files/zh-cn/learn/accessibility/index.md:41]:
```suggestion
  - : 这篇文章开头很好地阐述了到底什么是无障碍——这包括我们要考虑哪一群人，为什么要考虑这些人，不同用户使用什么工具与网页交互，以及我们如何在 web 开发工作流中构建无障碍部分。
```

**Review by yin1999 — APPROVED** [review]

**Pull request merged**